charts/pega/charts/installer/config/prconfig.xml.tmpl

```diff
@@ -45,4 +45,5 @@
   <env name="install/parallel/threadcount" value="-1" />
   <env name="install/parallel/index/threadcount" value="-1" />
   <env name="compiler/javasourcelevel" value="1.7" />
+  <env name="prconfig/cluster/hazelcast/v4/enabled" value="true" type="java.lang.String"/>
```

Comment:
If we are setting in prconfig.xml then we should not add prconfig to config name it should be <env name="cluster/hazelcast/v4/enabled" value="true" type="java.lang.String"/>, same applies for all the prconfig files